src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -245,6 +247,8 @@ protected OptionParser buildOptionParser() {
 		parser.accepts( "gtid_mode", "(experimental) enable gtid mode" ).withOptionalArg();
 		parser.accepts( "ignore_producer_error", "Maxwell will be terminated on kafka/kinesis errors when false. Otherwise, those producer errors are only logged. Default to true" ).withOptionalArg();
 		parser.accepts( "recapture_schema", "recapture the latest schema" ).withOptionalArg();
+		parser.accepts( "schema_chain_length", "length of schema chain after compaction" ).withRequiredArg();
```

Comment:
let's just make this 1.

Reply:
(or whatever means, compact all the way down to 1 schema.)

Reply:
Ok, I've removed this option and the whole chain always gets removed.